releasenotes/notes/sk-model-851851e42be88cc4.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    Introduced the Sherrington-Kirkpatrick (SK) model :class:`qiskit_optimization.applications.SKModel`.
```

Comment:
So in case you haven't seen these files are collected and form the release notes that end up here https://qiskit.org/documentation/optimization/release_notes.html. I see you had a slightly longer one with an example. I think many users may not know what this is and so at least having a reference link here and perhaps a sentence or two describing what you use this application/model for would be helpful. I saw you had an example - that can work too if you think its informative for a user. How about a tutorial on this? There the example can be runnable along with more description/information to help users understand/use the code etc.

Reply:
Thank you. Is there a way how to locally display the release notes?

Reply:
Nevertheless, I added there longer text with reference to a review paper. Regarding tutorial, I talked with Stefan and we leave it for another PR.

Reply:
@woodsp-ibm thank you. Please check the release note. If it is ok for you.

Reply:
Release note looks good to me. I did make a couple of other comments though in looking at the documentation that comes out for the class itself.